### ISSUE TITLE
d() crashes with segmentation fault 

### ISSUE THREAD

**Issue by THccaa** [bodkan/admixr#69]
Hi, 

I ran into a problem when using admixr to calculate D.
Admixr works fine with the example data from the tutorial vignette, but when I use my own data I get this error:

> Error: The output file we got from ADMIXTOOLS is truncated. Please examine the full output above and check for errors.

The output of the R console is:
>Segmentation fault
>BEGINNING OF OUTPUT FILE
>
>qpDstat: parameter file: res//qpDstat__1121120816.par
>###THE INPUT PARAMETERS
>##PARAMETER NAME: VALUE
>genotypename: ./Zoharim.geno
>snpname: ./Zoharim.snp
>indivname: ./Zoharim.ind
>popfilename: res//qpDstat__1121120816.pop
>printsd: YES
>##qpDstat version: 970
>
>END OF OUTPUT FILE
> 
>Error: The output file we got from ADMIXTOOLS is truncated.
>       Please examine the full output above and check for errors.

Since it works fine with the sample data, I thought that something is wrong with my data. However, I cannot find a mistake. I think the files are exactly like described in the vignette. 

`head Zoharim.geno `
>0000000100000900000000000000000010000000000000900000000000000000090000000
>0000000000000900000000000000000000000000000000000000000000000000090000000
>0000012000091900012000090000001000000100000010000002000001100000090000000
>0000000900000990000090090000000000000000000000000090009000000000990090000
>0000000000000900000000000000000000000001000000000000100000000000090000000

`head Zoharim.ind `
>Psoo14	U	sativum
>Psoo22	U sativum
>Psoo27	U	sativum
>PhZm01	U	s_humile
>PhZm09	U	s_humile
>PfZm28	U	fulvum
>Psoo42	U	sativum
>VL1	U	Outgroup

`head Zoharim.snp `
>8:12:+	1	0	13818	G	C
>8:26:+	1	0	13832	C	G
>8:91:+	1	0	13897	T	C
>8:106:+	1	0	13912	T	A
>8:122:+	1	0	13928	T	G
>11:36:-	1	0	13969	G	T
>78:9:-	1	0	39701	C	T

I thought that admixtools maybe builds blocks for the jack knife procedure based on genetic rather than physical distance and that the zeros cause the error. But I replaces the genetic distances in the sample .snp file with zeros and it yields the same results. So, that's not it.

These are the R commands that I run (pretty much the same like in the vignette):
`data_prefix <- "./Zoharim" `
`snps <- eigenstrat(data_prefix)`
`result <- d(W = "s_humile", X = "sativum", Y = "fulvum", Z = "Outgroup", data = snps, outdir = 'res/')`

Here some more output:
`cat ./res/qpDstat__1121120816.log`
>qpDstat: parameter file: res//qpDstat__1121120816.par
>###THE INPUT PARAMETERS
>##PARAMETER NAME: VALUE
>genotypename: ./Zoharim.geno
>snpname: ./Zoharim.snp
i>ndivname: ./Zoharim.ind
>popfilename: res//qpDstat__1121120816.pop
>printsd: YES
>##qpDstat version: 970

`cat ./res/qpDstat__1121120816.par`
>genotypename: ./Zoharim.geno
>snpname: ./Zoharim.snp
>indivname: ./Zoharim.ind
>
>popfilename: res//qpDstat__1121120816.pop
>
>printsd: YES

`cat ./res/qpDstat__1121120816.pop`
>s_humile sativum fulvum Outgroup

I would really appreciate some help, because I am absolutely clueless how to solve this issue. Thanks.

**Comment by bodkan** [comment]
Hi!

First of all, thank you for this wonderful and extensive bug report! This is truly exemplary and makes it possible to go right down to business. 

Segmentation fault indicates that the problem is most likely not with admixr, but with ADMIXTOOLS.

Given that you already have saved the `par` file, could you try running "raw" ADMIXTOOLS and paste the full output on the command-line? Just to make sure the same thing happens even without admixr.

Something like this:

```
qpDstat -p res/qpDstat__1121120816.par
```

Thank you.

**Comment by THccaa** [comment]
Thanks you @bodkan for the quick reply.
I already found the mistake, it was in fact a problem with my data. 
My vcf file contains super-scaffolds. I got an error before, that the chromosome names need to be integers. I changed the names of the super-scaffolds to be integers but they still had names like '1196' or '25160'. Apparently ADMIXTOOLS cannot handle such big numbers, because admixr works just fine when I exclude the super-scaffolds and run the analysis only with chromosomes 1-7.
